Assets/Src/AuthController/REST/REST Request Proxies/GoogleRefreshTokenRequestDtoProxy.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Src.AuthController.REST.REST_Request_Proxies
+{
+    [Serializable]
+    public class GoogleRefreshTokenRequestDtoProxy
+    {
+        [JsonProperty("client_id")]
+        public string clientID;
+
+        [JsonProperty("client_secret")]
+        public string clientSecret;
```

Comment:
You should avoid using public fields if possible and use public properties instead. Conversion to JSON should work fine with public automatic properties.